R/modelEstimation.R

```diff
@@ -27,24 +29,32 @@
 modelEstimation<-function(eList, 
                           windowY=7, windowQ=2, windowS=0.5,
                           minNumObs=100,minNumUncen=50, 
-                          edgeAdjust=TRUE){
+                          edgeAdjust=TRUE, verbose = TRUE, interactive = NULL){
 
+  if(!is.null(interactive)) {
+    message("The argument 'interactive' is deprecated. Please use 'verbose' instead")
+    verbose <- interactive
+  }
+  
   eList <- setUpEstimation(eList=eList, windowY=windowY, windowQ=windowQ, windowS=windowS,
                   minNumObs=minNumObs, minNumUncen=minNumUncen,edgeAdjust=edgeAdjust)
 
-  cat("\n first step running estCrossVal may take about 1 minute")
+  if(verbose) cat("\n first step running estCrossVal may take about 1 minute")
   Sample1<-estCrossVal(length(eList$Daily$DecYear),eList$Daily$DecYear[1],
                        eList$Daily$DecYear[length(eList$Daily$DecYear)], 
                        eList$Sample, 
                        windowY=windowY, windowQ=windowQ, windowS=windowS,
-                       minNumObs=minNumObs, minNumUncen=minNumUncen,edgeAdjust=edgeAdjust)
+                       minNumObs=minNumObs, minNumUncen=minNumUncen,edgeAdjust=edgeAdjust,
+                       verbose=verbose)
   
   eList$Sample <- Sample1
   
-  cat("\nNext step running  estSurfaces with survival regression:\n")
+  if(verbose) cat("\nNext step running  estSurfaces with survival regression:\n")
```

Comment:
I'm generally leaving in the `cat` when it's wrapped in `verbose`, since you can turn on/off the text with verbose, and I know the formatting is ever so slightly different between `cat` and `message`. So, if it's standalone, I'm using `message`, if it's via `verbose`, then `cat`. That being said, I could *easily* have missed a few